pkg/kubelet/server/stats/summary.go

```diff
@@ -87,3 +89,32 @@ func (sp *summaryProviderImpl) Get(updateStats bool) (*statsapi.Summary, error)
 	}
 	return &summary, nil
 }
+
+func (sp *summaryProviderImpl) GetCPUAndMemoryStats() (*statsapi.Summary, error) {
+	summary, err := sp.Get(false)
```

Comment:
Will it be hard to actually bypass getting the imageFS (and other unused) stats? Setting them to nil after getting all the data seems sub-optimal...

Reply:
Do you want me to bypass getting all unneeded stats or only some of them? Do you want me to add functions to StatsProvider that returns only needed stats?

Reply:
> Do you want me to bypass getting all unneeded stats or only some of them?

Bypassing all unneeded if possible.

> Do you want me to add functions to StatsProvider that returns only needed stats?

If that's needed, then yes :-)

Reply:
I have final https://github.com/kubernetes/kubernetes/pull/68841 but it is much bigger than this one.
Could we submit this PR and later submit final version? I would like to push this PR to 1.12 patch release and https://github.com/kubernetes/kubernetes/pull/68841 will be released with 1.13. It will be easier to release smaller PR in patch release.